src/probes/README.md

```diff
@@ -0,0 +1,53 @@
+# Probes
+
+The default probes provided along the telemetry client code are:
+
+* bertprobe: this probe reports on the Boot Error Region Table if such
+  entry is found in ```/sys/firmware/acpi/tables/BERT```.
+
+* crashprobe: this probe collects the core dump file from the kernel. This
+  probe is meant to be 'configured' in ```/proc/sys/kernel/core_pattern```.
+
+* hprobe: a simple probe that sends a keep alive message.
+
+* journalprobe: a probe for the journal.
+
+* oopsprobe: a probe to collect 'oops messages' when the kernel detects a
+  problem.
+
+* pstoreprobe: probe to collect messages left on pstore filesystem.
+
+
+### telem-record-gen
+
+Telemetry record generator is not exactly a probe, this tool can be used to
+implement probes i.e. using a bash script.
+
+For example to generate an hprobe like message we can execute the following
+command:
+
+```
+$ telem-record-gen --class aprobe/hello/world --payload hello
```

Comment:
update classification example according to revised classification description (above).